README.md

```diff
@@ -13,72 +13,101 @@ To get business, simply:
 $ gem install business
 ```
 
+## Important: 2.0.0 breaking changes
+
+We have removed the bundled calendars as of version 2.0.0, if you need the calendars that were included:
+
+Download the calendars you wish to use from [before version 2](https://github.com/gocardless/business/tree/b12c186ca6fd4ffdac85175742ff7e4d0a705ef4/lib/business/data) and place them in a suitable place in your project.
+
+Then, add the directory to where you placed the yml files before you load the calendar:
+
+```ruby
+Calendar::Business.load_paths("lib/calendars") # your_project/lib/calendars/ contains bacs.yml
+Calendar::Business.load("bacs")
+```
+
 ### Getting started
 
-Get started with business by creating an instance of the calendar class,
-passing in a hash that specifies with days of the week are considered working
-days, and which days are holidays.
+Get started with business by creating an instance of the calendar class, passing in a hash that specifies with days of the week are considered working days, and which days are holidays.
 
 ```ruby
 calendar = Business::Calendar.new(
   working_days: %w( mon tue wed thu fri ),
   holidays: ["01/01/2014", "03/01/2014"]    # array items are either parseable date strings, or real Date objects
+  extra_working_dates: [nil], # Makes the calendar to consider a weekend day as a working day.
 )
 ```
 
-`extra_working_dates` key makes the calendar to consider a weekend day as a working day.
+### Load a calendar from a file
 
-A few calendar configs are bundled with the gem (see [lib/business/data]((lib/business/data)) for
-details). Load them by calling the `load` class method on `Calendar`. The
-`load_cached` variant of this method caches the calendars by name after loading
-them, to avoid reading and parsing the config file multiple times.
+#### Calendar file definition
 
-```ruby
-calendar = Business::Calendar.load("weekdays")
-calendar = Business::Calendar.load_cached("weekdays")
-```
+Defining a calendar as a Ruby object may not be convient, to load it from a YAML file follow and customise the example below. All keys are optional and will default to the following:
```

Comment:
```suggestion
Defining a calendar as a Ruby object may not be convenient, `business` also allows you to define a calendar as a YAML file and load this into your application.

Below we will walk through the necessary steps to build your first calendar.
```

Can we follow the thread of this and walk through the creation of a calendar?

Reply:
👍